apps/wallet/src/ui/app/pages/home/nft-transfer/TransferNFTForm.tsx

```diff
@@ -40,7 +40,18 @@ function TransferNFTForm({
     useEffect(() => {
         onClearRef.current();
     }, [to]);
-    const maxGasCoinBalance = useIndividualCoinMaxBalance(SUI_TYPE_ARG);
+
+    const accountAddress = useAppSelector(({ account }) => account.address);
+    const { data: coinBalance, isLoading: loadingBalances } = useGetCoinBalance(
+        SUI_TYPE_ARG,
+        accountAddress
+    );
+
+    const maxGasCoinBalance = useMemo(
```

Comment:
Doesn't need to be memo'd. We should only pull this out to optimize known expensive computation or to ensure referential stability for props.